packages/parser/src/schemas/api-gateway-websocket.ts

```diff
@@ -0,0 +1,90 @@
+import { z } from 'zod';
+
+/**
+ * A zod schema for API Gateway Proxy WebSocket events.
+ *
+ * @example
+ * {
```

Comment:
The example code should be wrapped within three backticks, see example [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/e798a703424dcc976ae2d9a82790e0f2c611510b/packages/parser/src/schemas/api-gatewayv2.ts#L21).